notimetowait2.md

```diff
@@ -14,6 +14,8 @@ Event Hashtag: **#notimetowait**
 
 Hosted by the [Österreichisches Filmmuseum - Austrian Film Museum](https://www.filmmuseum.at) and [MediaArea.net](https://mediaarea.net)
 
+Locally organized by [AV-RD](http://www.av-rd.com) and social dinner sponsored by [ANYFER](http://www.anyfer.com)
```

Comment:
but when?
Also from the phrasing it's unclear what is locally organized by AV-RD: the entire symposium is the dinner.

Reply:
I didn't want to put the "technical" (i.e. date) details there, only listing involved people.
I don't see the unclear part, there is something before the 'and' and something after.
Suggestion?